Project.toml

```diff
@@ -13,7 +13,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Adapt = "2, 3.2"
-ChainRulesCore = "0.9.45, 0.10, 1"
+ChainRulesCore = "0.9.45, 0.10, 1-1.17"
```

Comment:
```suggestion
ChainRulesCore = "0.9.45, 0.10, 1 - 1.17"
```

Reply:
Oh sorry, was reading the docs while you were writing.

Reply:
Oh I'm an idiot, that bounds the wrong package, sorry

Reply:
OK, restricting CR fixes it. I believe the cause is https://github.com/JuliaDiff/ChainRules.jl/pull/559/files#diff-7a31ac47520a0ad235f15cead3f4c48fcb595dc7f2421ebbee8ffce23e2510f5R84 , probably repaired in https://github.com/JuliaDiff/ChainRules.jl/pull/562

Reply:
ok, seems fixed!